app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -158,6 +158,7 @@
         with_border: !omit_footer_border,
         navigation: omit_footer_navigation ? nil : layout_helper.footer_navigation,
         meta: layout_helper.footer_meta,
+        ga4_index_total: layout_helper.get_footer_links_count
```

Comment:
This passes the amount of footer links down to where the OGL and Crown Copyright links are hardcoded, so that they can access the value.